test/src/main/java/org/apache/accumulo/test/ImportExportIT.java

```diff
@@ -119,23 +126,25 @@ public void testExportImportThenScan() throws Exception {
       FSDataInputStream is = fs.open(distcp);
       BufferedReader reader = new BufferedReader(new InputStreamReader(is));
 
-      // Copy each file that was exported to the import directory
+      // Copy each file that was exported to one of the imports directory
       String line;
+
       while ((line = reader.readLine()) != null) {
         Path p = new Path(line.substring(5));
         assertTrue("File doesn't exist: " + p, fs.exists(p));
-
+        Path importDir = importDirAry[r.nextInt(importDirAry.length)];
         Path dest = new Path(importDir, p.getName());
         assertFalse("Did not expect " + dest + " to exist", fs.exists(dest));
         FileUtil.copy(fs, p, fs, dest, false, fs.getConf());
       }
 
       reader.close();
 
-      log.info("Import dir: {}", Arrays.toString(fs.listStatus(importDir)));
+      log.info("Import dir A: {}", Arrays.toString(fs.listStatus(importDirA)));
```

Comment:
Does this test create more than one file for export?  When you run this test do you see files for each dir?

Reply:
Unfortunately not. I need to extend the test to properly do this.

Reply:
Maybe adding some splits to the table and compacting would be a simple way to achieve this.

Reply:
@drewfarris  was this resolved?  Did a past version of the test only have a single file?  There are two files now it seems, are those on different volumes?